templates/single.twig

```diff
@@ -91,6 +91,19 @@
 						{{ fn('do_blocks', post.take_action_boxout )|raw }}
 					{% endif %}
 				</article>
+				{% if ( post.tags ) %}
+					{% for tag in post.tags %}
+						<a
+							class="btn btn-post-tag"
+							href="{{ tag.link }}"
+							data-ga-category="Post"
+							data-ga-action="Navigation Tag"
+							data-ga-label="n/a"
```

Comment:
`data-ga-label` is [not required](https://www.hotjar.com/google-analytics/glossary/events/). We can probably exclude the data attribute instead of putting `n/a` everywhere.

But I didn't check if it's really not required by our GA setup, perhaps it somehow crashes while parsing the data attributes if the value is not there.

If that's the case, we should update the analytics container code to not require it. Or maybe make it add `n/a` by default in case the attribute is missing.

Reply:
This should probably be done in a separate ticket, since we use this in a lot of other places, and we'd need to double-check with Julia and/or the new data analyst before making the changes 🤔

Reply:
You mean the part of updating the code in the GA container, right? Sure, that's not something I expect we can quickly change.

But we could already just confirm what happens if the `data-ga-label` is not present. I don't expect this to be a problem, in which case we can already exclude the useless `n/a` value.

Reply:
Well I meant more the part about updating all the places where we use this `n/a` value, I don't think we should do this as part of this PR

Reply:
No indeed, I just meant to exclude it from then new code if we find it doesn't need to be there.

Reply:
[Removed](https://github.com/greenpeace/planet4-master-theme/pull/1630/commits/ddf32c299e0ef4104034f993e22140546c09fb0c)